superset-frontend/src/explore/components/DatasourcePanel.tsx

```diff
@@ -182,13 +183,13 @@ const DataSourcePanel = ({
         actions={actions}
         formData={datasourceControl.mapStateToProps}
       />
+      <input
+        type="text"
+        onChange={search}
+        className="form-control input-sm"
+        placeholder={t('Search Metrics & Columns')}
+      />
```

Comment:
Can we use AntD input here？The style look too bootstrap-ish.

Reply:
Let's follow up on that in a separate PR. We should convert ALL inputs to AntD inputs, and just make a big sweep of 'em.